Jenkinsfile

```diff
@@ -48,6 +48,7 @@ pipeline {
               agent any
 
               environment {
+                APP_ENV = "${env.TAG_NAME == "production-release" ? "production" : "staging"}"
```

Comment:
sadly TAG_NAME will never be available here as it's under the master branch `when { branch 'master' }`